src/index.html

```diff
@@ -26,5 +26,10 @@ <h1 class="d-flex space-between p-2 title">Today's To Do <span class="fa fa-refr
         </div>
       </section>
     </main>
+    <!--
+      <footer>
+      <div>Developed by &copy; Felix Ouma</div>
+    </footer>
+  -->
```

Comment:
- [ ] Avoid leaving commented-out code in the source code as it is bad practice, it takes up space, causes confusion, and leads to maintenance issues if it's not removed